pkg/controller/provider/controller.go

```diff
@@ -131,72 +126,57 @@ var _ reconcile.Reconciler = &Reconciler{}
 //
 // Reconciles an provider object.
 type Reconciler struct {
-	record.EventRecorder
-	client.Client
+	base.Reconciler
 	catalog   *Catalog
-	scheme    *runtime.Scheme
 	container *libcontainer.Container
 	web       *libweb.WebServer
-	log       *logging.Logger
 }
 
 //
 // Reconcile a Inventory CR.
 // Note: Must not a pointer receiver to ensure that the
 // logger and other state is not shared.
 func (r Reconciler) Reconcile(request reconcile.Request) (result reconcile.Result, err error) {
-	fastReQ := reconcile.Result{RequeueAfter: FastReQ}
-	slowReQ := reconcile.Result{RequeueAfter: SlowReQ}
-	noReQ := reconcile.Result{}
-	result = noReQ
-
-	r.log = logging.WithName(
+	r.Log = logging.WithName(
 		names.SimpleNameGenerator.GenerateName(Name+"|"),
-		"provider",
+		"hook",
```

Comment:
This looks like a probable cut and paste error, hook -> provider

Reply:
Fixed.